src/app/components/apps-view/apps-view.component.ts

```diff
@@ -79,6 +79,7 @@ export class AppsViewComponent implements OnInit {
       { colId: 'applicationId', colName: 'Application ID' },
       { colId: 'queueName', colName: 'Queue Name' },
       { colId: 'applicationState', colName: 'Application State' },
+      { colId: 'lastStateChangeTime', colName: 'Last State Change Time' },
```

Comment:
This could be like:
```
      {
        colId: 'lastStateChangeTime',
        colName: 'Last State Change Time',
        colFormatter: CommonUtil.timeColumnFormatter,
      },
```

Reply:
Same for submissionTime too.
```
{
        colId: 'submissionTime',
        colName: 'Submission Time',
        colFormatter: CommonUtil.timeColumnFormatter,
      },

```

Reply:
updated.